### DISCUSSION TITLE
Fully AFK Paragon Farming

### DISCUSSION THREAD

**Discussion by acmihal** [kitten-science/kitten-scientists#103] [Ideas]
I have a configuration that will do completely unattended paragon farming. If you load the attached savegame and unpawse it should take a minute to get up to 25k unob and then automatically reset. It can be left alone and will generate 600 paragon about every 90 minutes. I've had it running for about two days with 2.0.0-alpha.10-live and it seems stable.

There are a couple issues that could get the cycle time down to around 30 minutes:

1. Timeskip to Redmoon. This is dependent on #19 to unlock Chronoforge and #18 to skip to Redmoon without waiting for chronoheat cooldown or needing to buy chronofurnaces.
2. Limit eludium crafting to about 150. You only need enough to buy Microwarp Reactors after the reset. The limit is currently not persistent across resets and this will be fixed by #16.
3. #14 will speed things up by getting Solar Revolution as fast as possible after reset.
4. #17 could help a little but is probably not that important.

The configuration assumes you are at the point of the game where you have Renaissance, Anachronomancy, Pawgan Rituals, and Numeromancy. It also may have some dependence on the specific set of cryptos that I have. Some tuning may be necessary.

The important details about the configuration are:

1. I'm going for reset at 25k unob and 5 chronospheres based on Math Hacks http://www.bloodrizer.ru/games/kittens/wiki/index.php?page=Math+Hacks
2. Most Bonfire buildings are limited so that resources will saturate and carry over with Flux Condensator.
3. In Village, most kittens are assigned to Geologist to maximize gold for trading with zebras for titanium.
4. In Science->Technologies, anything that costs relics is turned off. We only want to spend relics on Tachyon Theory and Chronoforge.
5. Science->Policies is set up to maximize unob production and kitten arrival rate.
6. Workshop should have a max eludium of 150, but this is currently not persistent.
7. The Trading trigger is set to 0.03 instead of the default 0.98 to get zebra trading going faster.
8. In Religion, only do Solar Revolution and skip all the other Order of the Sun items.
9. In Space, only do the Dune, Moon, Orbital, and Piscine missions. Make everything except for Moon Bases, Spice Refineries, and Orbital Arrays. You may need Moon Bases when you are getting started in order to have at least 25k unob for reset, but the attached savegame is past that point.
10. In Time Skip, skip every cycle except Redmoon and skip in any season.
11. In Reset Timeline (Danger!), the only condition you need to activate is Unob >= 25k. You also want to have 5 chronospheres and Flux Condensator and 150 eludium before reset, but I think you are guaranteed to have these by the time unob gets up to 25k anyway.

Comments and suggestions appreciated. I hope you find this useful.

[Kittens Game - Run 98 - Year 6 - Autumn, day 29.txt](https://github.com/kitten-science/kitten-scientists/files/10019379/Kittens.Game.-.Run.98.-.Year.6.-.Autumn.day.29.txt)

**Comment by oliversalzburg** [comment]
I really love how many aspects of the script are active for this to work. I'll do my best to work on the open issues :)

**Comment by acmihal** [comment]
[20221206_fast_paragon_settings.txt](https://github.com/kitten-science/kitten-scientists/files/10376241/20221206_fast_paragon_settings.txt)
Here are my current KS-only settings for the fast paragon run. The savegame attached before apparently has the game state but not the KS settings stored in it.

**Reply by oliversalzburg** [reply]
I feel like a lot of things you mentioned in your post are not reflected in those settings.
